src/sentry/static/sentry/app/views/eventsV2/table/queryField.tsx

```diff
@@ -362,8 +374,10 @@ class QueryField extends React.Component<Props> {
       },
     };
 
+    const parameters = this.renderParameterInputs(parameterDescriptions);
+
     return (
-      <Container className={className} gridColumns={gridColumns}>
+      <Container className={className} gridColumns={parameters.length + 1}>
```

Comment:
Should the `gridColumns` property be renamed to `columns`?

Reply:
Yeah, I'll deal with that in a sep PR I think.